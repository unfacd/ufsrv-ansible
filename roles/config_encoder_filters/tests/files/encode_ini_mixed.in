var1: some text
var2: 123

section1:
  var3: 987.654

section2:
  var4: yes
  var5: "false"
