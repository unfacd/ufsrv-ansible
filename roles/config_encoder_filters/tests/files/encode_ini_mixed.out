var1=some text
var2=123

[section1]
var3=987.654

[section2]
var4=True
var5=false
